composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "codezero/laravel-localized-routes",
+    "name": "skyraptor/laravel-localized-routes",
```

Comment:
Is it nornal to change the package name here? (or a merge mistake?)

Reply:
Yeah that happens when you merge using GitHub rather than cli